frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelStepsPicker.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { funnelLogic } from 'scenes/funnels/funnelLogic'
+import { FunnelVizType } from '~/types'
+import { Row, Select } from 'antd'
+import { ANTD_TOOLTIP_PLACEMENTS } from 'lib/utils'
+
+export function FunnelStepsPicker(): JSX.Element | null {
+    const { filters, numberOfSeries, areFiltersValid } = useValues(funnelLogic)
+    const { changeHistogramStep } = useActions(funnelLogic)
+
+    if (filters.funnel_viz_type === FunnelVizType.Steps) {
+        return null
+    }
+
+    const onChange = (funnel_from_step?: number, funnel_to_step?: number): void => {
+        changeHistogramStep(funnel_from_step, funnel_to_step)
+    }
+
+    return (
+        <Row align="middle" style={{ margin: '8px 0' }}>
+            Between
```

Comment:
I think @mariusandra and @neilkakkar have a point. How about renaming this from just "Between X and Y" to:
- for Time to convert: "Conversion time from Step X to Step Y",
- for Historical trends: "Conversion rate from Step X to Step Y"?